Patches/Rim Flood/CE_FloodForms.xml

```diff
@@ -0,0 +1,464 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+	<Operation Class="PatchOperationFindMod">
+		<mods>
+		<li>Rim Flood</li>
+		</mods>
+
+		<match Class="PatchOperationSequence">
+			<operations>
+
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[@Name="BaseFlood"]/statBases</xpath>
+				<value>
+					<SmokeSensitivity>0.05</SmokeSensitivity>
+				</value>
+			</li>
+
+			<li Class="PatchOperationAddModExtension">
+				<xpath>/Defs/ThingDef[defName="InfestForm" or defName="Combatform" or defName="FloodCarrier" or defName="Tankform"]</xpath>
+				<value>
+					<li Class="CombatExtended.RacePropertiesExtensionCE">
+						<bodyShape>Humanoid</bodyShape>
+					</li>
+				</value>
+			</li>
+
+			<li Class="PatchOperationAddModExtension">
+				<xpath>/Defs/ThingDef[defName="Houndform" or defName="Pureform"]</xpath>
+				<value>
+					<li Class="CombatExtended.RacePropertiesExtensionCE">
+						<bodyShape>Quadruped</bodyShape>
+					</li>
+				</value>
+			</li>
+
+			<li Class="PatchOperationAddModExtension">
+				<xpath>/Defs/ThingDef[defName="Stalkerform"]</xpath>
+				<value>
+					<li Class="CombatExtended.RacePropertiesExtensionCE">
+						<bodyShape>QuadrupedLow</bodyShape>
+					</li>
+				</value>
+			</li>
+
+		<li Class="PatchOperationReplace">
+			<xpath>Defs/ThingDef[defName="Houndform"]/statBases/MoveSpeed</xpath>
+			<value>
+				<MoveSpeed>6.2</MoveSpeed>
+			</value>
+		</li>
+
+		<li Class="PatchOperationReplace">
+			<xpath>Defs/ThingDef[defName="Combatform"]/statBases/MoveSpeed</xpath>
+			<value>
+				<MoveSpeed>4.0</MoveSpeed>
+			</value>
+		</li>
+
+		<li Class="PatchOperationReplace">
+			<xpath>Defs/ThingDef[defName="FloodCarrier"]/statBases/MoveSpeed</xpath>
+			<value>
+				<MoveSpeed>1.5</MoveSpeed>
+			</value>
+		</li>
+
+		<li Class="PatchOperationReplace">
+			<xpath>Defs/ThingDef[defName="Tankform"]/statBases/MoveSpeed</xpath>
+			<value>
+				<MoveSpeed>3.0</MoveSpeed>
+			</value>
+		</li>
+
+		<li Class="PatchOperationReplace">
+			<xpath>/Defs/ThingDef[defName="Combatform"]/statBases/ArmorRating_Blunt</xpath>
+			<value>
+				<ArmorRating_Blunt>4</ArmorRating_Blunt>
+			</value>
+		</li>
+		<li Class="PatchOperationReplace">
```

Comment:
In order to avoid making them overly armored against lower-tech melee (as bad an idea as melee is against the flood anyway), I might format them much like insects--lower armor with limited coverage, but higher health and basically no pain shock level.

Reply:
Half of them already use insect bodies, the ones that use the flood body type don't have limbs, mostly just head and body.

Reply:
In Vanilla most of the flood creatures are made to have 1-3 hp at most, so any hit are instant kill, probably due to vanilla ranged being rubbish.

Reply:
Strange, flood carriers are patched in the exact same way as the other flood forms. I don't notice any difference in the xml structure of it and the other flood forms, is it just the flood carriers not working, or is it all the other flood forms as well?

Reply:
Forerunners, like the flood, seems to be weak in the original mod, I think they are meant to be weak as the author want the forerunners and flood kinda annihilate eachother on contact,
 I don't really want to change the balance too much.